src/cmd/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ if(NOT BUILD_LIBRARIES_ONLY)
 	set_target_properties(${cmd_NAME} PROPERTIES
 	        RUNTIME_OUTPUT_DIRECTORY  ${BIN_OUTPUT_DIRECTORY} )
         set_target_properties(${cmd_NAME} PROPERTIES
-		INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/${LIB_INSTALL_DIR}/${APPLICATION_EXECUTABLE}" )
+		INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/${LIB_INSTALL_DIR}" )
```

Comment:
I am not sure if we have to install an rpath at all if a standard lib path is used. I think rpath is only neccessary if we ship the lib in a private dir.